README.md

```diff
@@ -54,12 +54,13 @@ The binding provides a `Whisper` class:
 from whispercpp import Whisper
 
 w = Whisper.from_pretrained("tiny.en")
+w = Whisper.from_pretrained("/path/to/ggml-model.bin")
```

Comment:
let's keep the upper API docs to have the default alias.

Reply:
Okay! I added a simple comment down in the api docs now and removed this one here.